docs/release-notes/v1.50-v1.59.md

```diff
@@ -2,13 +2,310 @@
+    * Upon calling the endpoint
+        * An attachment object is created in ODM.
+        * The reference between the specified study and the created object is created
+    * Request body:
+        * The <code>source</code> field is optional. If no source is provided, the default value is used. Allowed values for source are "S3", "HTTP", and "LOCAL".
+        * The <code>dataLink</code> field is mandatory and must contain the link to the file to be attached.
+        * The <code>dataClass</code> field is mandatory and it contains the data type (e.g. transcriptomics, lipidomics, etc.)
+        * The <code>metadataLink</code> field is optional and allows users to provide a metadata file to add context, which will also be loaded into ODM.
+        * The <code>studyAccession</code> field is mandatory and must specify the ID of the study to which the file will be attached
+    * Additional notes:
+        * Unlike the GUI, the API does not impose a 5GB size limit for file uploads. This feature is only available to customers with a configured S3 bucket for storage.
+* ??? danger "Endpoint to import Files"
+    * POST /api/v1/jobs/import/file
+    ![ImportUpdated.png](1.59/ImportUpdated.png){width=800}
+    <figcaption>The newly introduced <code>POST /api/v1/jobs/import/file</code> endpoint, users can efficiently import file attachments via APIs</figcaption>
+
+* **Improved Support for Files (formerly known as Attachments): Import files and assign Data Class via GUI**
+
+To complement the API improvements, the graphical user interface (GUI) now supports assigning a **Data Class** when uploading files. This enhancement ensures that non-technical users can achieve the same level of organization without relying on APIs
+
+* Enhancements in the GUI:  
+    * A Data Class dropdown has been added to the file attachment window.  
+    * Updated tooltip for the **Attach a file** button:  
+    * Add various research materials related to your study, such as PDFs, XLSX, DOCX, PPTX, images, and more.
+
+* ??? abstract "Important considerations"  
+    * All users with access to the study can download the attached files.  
+    * Unlike the **Import Data** option, the attached files’ content will not be indexed for search but can be located using their name and metadata.  
+    * The attached files are grouped by their Data Class and now include a **unique identifier (accession number)** for easier retrieval.
+
+![AttacahedAccession](1.59/AttacahedAccession.png){width=700}
+<figcaption>Import files via GUI, assign a data class, and view attached files in the Data tab, complete with their type and accession number</figcaption>
+
+* **Improved Support for Files (formerly known as Attachments): Improved Metadata Management for Attached Files**
+Managing metadata for attached files is now as straightforward as managing metadata for other entities in the system. This includes capabilities for browsing, updating, and searching metadata.
```

Comment:
https://genestack.atlassian.net/browse/ODM-12154 will be completed in 1.59, so we can mention that the metadata is editable as well.

Reply:
@MikhailAf fyi: this is still actual.

Reply:
Done